### PR TITLE
feat(settings/language): added possibility to set default language

### LINKED ISSUE #501: Enhancement: Standard language
### Application

Settings

### Dependency

No

### Feature Description

An option "DefaultLanguage" would be very nice in the config.json

### References

_No response_

### PR DESCRIPTION
**Pull request by antonstjernquist**
Allow default language to be set config

**Pull Request Checklist**:
* [ X] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [ X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ -] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "general": {
     "useResourceIntegration": false,
     "toggleKey": "f1",
-    "toggleCommand": "phone"
+    "toggleCommand": "phone",
+    "defaultLanguage": "sv"
```
Comment (antonstjernquist) [inline config.json:11]: This should probably be "en" by default.

File: phone/src/Phone.tsx [hunk 2]
```diff
@@ -72,7 +70,7 @@ function Phone() {
               <Route exact path="/" component={HomeApp} />
               {callModal && <Route exact path="/call" component={CallModal} />}
               {apps.map((App) => (
-                <>{!App.isDisabled && <App.Route key={App.id} />}</>
+                <Fragment key={App.id}>{!App.isDisabled && <App.Route key={App.id} />}</Fragment>
```
Comment (antonstjernquist) [inline phone/src/Phone.tsx:73]: Fixing key error in "Phone.tsx"

File: phone/src/os/phone/hooks/usePhone.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const usePhone = (): IUsePhone => {
   };
 
   return {
-    ResourceConfig: config as ResourceConfig,
```
Comment (antonstjernquist) [inline phone/src/os/phone/hooks/usePhone.ts:31]: Unnecessary convert here

File: phone/src/Phone.tsx (hunk 2 above)
Comment (Patilhas) [reply phone/src/Phone.tsx:73]: You may want to add this fix also in `ui/components/GridMenu.tsx`.

Comment (antonstjernquist) [reply phone/src/Phone.tsx:73]: Sure

File: config.json (hunk 1 above)
Comment (itschip) [reply config.json]: Yes :)

Comment (antonstjernquist) [reply config.json]: Fixed

**Review by itschip — APPROVED** [review]

**Pull request merged**